build.gradle

```diff
@@ -1,3 +1,19 @@
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" } // Mirrors jcenter() and mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.6'
+        classpath "com.gradle.publish:plugin-publish-plugin:0.11.0"
+        classpath "com.github.ben-manes:gradle-versions-plugin:0.12.0"
+    }
+}
+
+plugins {
+    id("org.gradle.kotlin.kotlin-dsl").version("1.4.9")
```

Comment:
The typical syntax is `id "org.gradle.kotlin.kotlin-dsl" version "1.4.9"`

Reply:
Changed it to the typical syntax.